src/reducers/computeRenderedRows.js

```diff
@@ -241,18 +235,15 @@ function computeRenderedRowOffsets(state, rowRange, viewportOnly) {
     return;
   }
 
-  const startIdx = viewportOnly ? firstViewportIdx : firstBufferIdx;
-  const endIdx = viewportOnly ? endViewportIdx : endBufferIdx;
```

Comment:
Backports the behavior to viewport buffering logic (#421) so that FDT will always update the entire row buffer instead of just updating no:of rows equal to the count of the viewport.